test/sqllogictest/transform/topk.slt

```diff
@@ -105,10 +106,12 @@ create default index on v1
 ----
 
 query T multiline
-explain with(arity, join_impls) view v1;
+explain with(arity, join_impls) select * from v1;
 ----
-materialize.public.v1:
-  TopK order_by=[#0 asc nulls_last] limit=3 offset=3 // { arity: 2 }
-    Get materialize.public.t1 // { arity: 2 }
+Explained Query (fast path):
+  ReadExistingIndex materialize.public.v1_primary_idx
```

Comment:
This demonstrates the problem with the suggestion of `select * from viewname`. We are unfortunately no longer testing the real plan here.